seedDB.js

```diff
@@ -0,0 +1,39 @@
+const db = require('./database/database.js');
+const bn = require('./bandNames.js');
+
+const apiKey = require('./config.js');
+const axios = require('axios');
+const bodyParser = require('body-parser');
+
+const seedDatabase = () => {
+  var imageData = [];
+  axios.get(`https://api.unsplash.com/search/photos?query=music&per_page=30&client_id=${apiKey}`)
+    .then((response) => {
+      console.log('Response from Image GET: ', response.data.results);
+      imageData.push(...response.data.results);
```

Comment:
ok for a first pass, tho you don't really need all that data. I suggest you only pass in the url. I don't want to bog you down, use your judgement, the images should be hosted on S3 or another instance you control, not the direct link to unsplash. 

Like I say, please don't let a refactor stop your progress. Use your judgement as to when/if you can do this refactor.